packages/core/src/storages/request_provider.ts

```diff
@@ -766,8 +776,12 @@ export abstract class RequestProvider implements IStorage {
         const queue = await manager.openStorage(queueIdOrName, options.storageClient);
         queue.proxyConfiguration = options.proxyConfiguration;
 
-        // eslint-disable-next-line dot-notation
-        queue['initialCount'] = (await queue.client.get())?.totalRequestCount ?? 0;
+        const queueInfo = await queue.client.get();
+
+        // eslint-disable dot-notation
```

Comment:
if this doesn't cause TS error (accessing `private` fields), I suppose we can use dot notation (already happening) and remove those `eslint-disable`s

Reply:
eslint actually makes you replace square bracket access with a dot, and the rule for that is called dot-notation. We need those comments 😁

Reply:
But there are no square brackets anymore, this PR removes them :sweat_smile: which is why I'm surprised that TS is not complaining about accessing private fields with dot, but those eslint comments shouldn't do anything anyway

btw those lines actually don't do anything anyway, for the `eslint-disable` directive to apply over multiple lines, it has to be in a `/* block comment */`. Line comments (`//`) only work with `eslint-disable-(next-)line`. I suppose that's why this is not marked as an unused disable comment.

![image](https://github.com/user-attachments/assets/ecde8883-c609-4064-89b5-8885c8272f3f)

Reply:
Oh, I see :facepalm: In the current state, the `eslint-disable` comments don't do anything and `eslint --fix` removed the square bracket access. And apparently that's fine so I'll just remove the comments.